### DISCUSSION TITLE
Add lyrics to tracks

### DISCUSSION THREAD

**Discussion by simonv3** [funmusicplace/mirlo#621] [Ideas]
This would likely be as simple as a field on a track that's just a `text` field. 

Then artists need a way to add lyrics to a track, and then lyrics need to be displayed on a track

**Comment by OutersiderOfficial** [comment]
I'm all for the idea of adding written lyrics, I look forward to see this come to fruition at Mirlo.

**Comment by LouisLouiseKay** [comment]
On album page, I think lyrics could appear in a modal for the sake of simplicity. And we need to figure out if we add a dropdown for "share track / lyrics" or if we add a small logo for lyrics next to the track (which might prompt more people to try to look at lyrics), this is how Bandcamp does it.

**Comment by simonv3** [comment]
This was implemented!
